pages/common/git-commit.md

```diff
@@ -1,6 +1,6 @@
 # git commit
 
-> Commit staged files to the repository.
```

Comment:
Why remove `staged` here?

Reply:
Since the new content describes how to commit files even before staging (auto stage and commit) them, I thought removing 'staged' in the original desc made more sense.

Reply:
Ah, ok. Sure!